### DISCUSSION TITLE
what is index of collection?

### DISCUSSION THREAD

**Discussion by ghost** [pocketbase/pocketbase#2899] [Q&A]
i want to know what a index in a collection is what can it be used for?

**Comment by ganigeorgiev** [comment]
A collection index statement serves the same purpose as SQLite index. Internally when you create an index SQLite will maintain its own "lookup/search" table with the indexed data, allowing faster queries on the searched column.

Another use of indexes could be to define unique constraints across one or multiple collection fields. For more details on the index syntax, you can refer to https://www.sqlite.org/lang_createindex.html.
